### PR TITLE
Improvments in README

### PR DESCRIPTION
**Pull request by scraly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -648,6 +845,17 @@ How to compile?
...
+
+Our awesome contributors:
+
+<a href="https://github.com/ovh/venom/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=ovh/venom" />
+</a>
+
 # License
 
 Copyright 2021 OVH SAS
```
Comment (yesnault) [inline README.md:861]:
```suggestion
Copyright 2022 OVH SAS
```

**Review by yesnault — CHANGES_REQUESTED** [review]

**Review by yesnault — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,38 +1,150 @@
...
+  * [Common errors with quotes](#common-errors-with-quotes)
+* [Use venom in CI/CD pipelines](#use-venom-in-ci-cd-pipelines)
 * [Hacking](#hacking)
+* [Contributing](#contributing)
 * [License](#license)
 
-# Command Line
+# Overview
+
+Venom allow you to handle integration test in the same way you code your application.
```
Comment (fsamin) [inline README.md:52]:
```suggestion
Venom allows you to handle integration tests the same way you code your application.
```

File: README.md [hunk 3]
```diff
@@ -1,38 +1,150 @@
...
+With Venom, test cases can be managed like code: the readability of the test description means that the tests are part of the code reviews. Thanks to that, write and execute testsuites become easier for developers and teams.
```
Comment (fsamin) [inline README.md:53]:
```suggestion
With Venom, testcases will be managed as code: the readability of the tests means that the tests are part of the code reviews. Thanks to that, write and execute testsuites become easier for developers and teams.
```

File: README.md [hunk 4]
```diff
@@ -1,38 +1,150 @@
...
+
+<img src="./venom.gif" alt="Venom Demonstration">
+
+# Installing
+
+## Install from binaries
+
+1. Download latest binary release from: https://github.com/ovh/venom/releases/latest/
+
+You can directly download a specific version for a specific environment, version 1.0.1 for Linux for example to a binary file named `venom`
```
Comment (richardlt) [inline README.md:67]:
```suggestion
Example for Linux:
```

File: README.md [hunk 5]
```diff
@@ -1,38 +1,150 @@
...
+Venom run executors (scripts, HTTP Request, web, IMAP, etc.) and apply assertions. 
+It can also generate xUnit result files.
+
+<img src="./venom.gif" alt="Venom Demonstration">
+
+# Installing
+
+## Install from binaries
+
+1. Download latest binary release from: https://github.com/ovh/venom/releases/latest/
```
Comment (richardlt) [inline README.md:65]:
```suggestion
You can find latest binary release from: https://github.com/ovh/venom/releases/latest/.
```

File: README.md [hunk 6]
```diff
@@ -1,38 +1,150 @@
...
+
+You can directly download a specific version for a specific environment, version 1.0.1 for Linux for example to a binary file named `venom`
+
+```bash
+$ curl -LO https://github.com/ovh/venom/releases/download/v1.0.1/venom.linux-amd64 venom
```
Comment (richardlt) [inline README.md:70]:
```suggestion
$ curl https://github.com/ovh/venom/releases/download/v1.0.1/venom.linux-amd64 -L -o /usr/local/bin/venom && chmod +x /usr/local/bin/venom
$ venom -h
```

File: README.md [hunk 7]
```diff
@@ -380,17 +528,9 @@ Builtin variables:
...
 * {{.venom.libdir}}
 
-# Tests Report
-
-```bash
-venom run --format=xml --output-dir="."
-```
-
-Available formats: jUnit (xml), json, yaml, tap reports
+## Assertion
```
Comment (fsamin) [inline README.md:531]:
```suggestion
## Assertions
```

File: README.md [hunk 8]
```diff
@@ -472,13 +612,71 @@ Supported operators are `and`, `or` and `xor`.
 
 More examples are available in [`tests/assertions_operators.yml`](/tests/assertions_operators.yml).
 
+# Write and run your first test suite 
+
+To try to understand how the Venom works, we will create and run a first testsuite together.
```
Comment (fsamin) [inline README.md:617]:
```suggestion
To understand how the Venom is working, let's create and run a first testsuite together.
```

File: README.md [hunk 9]
```diff
@@ -1,38 +1,150 @@
...
+
+1. Download latest binary release from: https://github.com/ovh/venom/releases/latest/
+
+You can directly download a specific version for a specific environment, version 1.0.1 for Linux for example to a binary file named `venom`
+
+```bash
+$ curl -LO https://github.com/ovh/venom/releases/download/v1.0.1/venom.linux-amd64 venom
+```
+
+2. Make binary executable
```
Comment (richardlt) [inline README.md:73]: Remove step 2 to 4.

File: README.md [hunk 10]
```diff
@@ -472,13 +612,71 @@ Supported operators are `and`, `or` and `xor`.
 
 More examples are available in [`tests/assertions_operators.yml`](/tests/assertions_operators.yml).
 
+# Write and run your first test suite 
+
+To try to understand how the Venom works, we will create and run a first testsuite together.
+
+The first test that we will do, in this testsuite, is to test whether the site we want to test (a public REST API, OVHcloud API for example):
+- is accessible (respond with a 200 status code)
+- respond quickly (less than 1 second)
```
Comment (fsamin) [inline README.md:621]:
```suggestion
- respond in less than 5 seconds
```

**Review by fsamin — CHANGES_REQUESTED** [review]
Thank you very much for your contribution, here are some suggestions.

File: README.md (hunk 8 above)
Comment (richardlt) [inline README.md]:
```suggestion
To understand how the Venom works, we will create and run a first testsuite together.
```

File: README.md [hunk 11]
```diff
@@ -472,13 +612,71 @@ Supported operators are `and`, `or` and `xor`.
 
 More examples are available in [`tests/assertions_operators.yml`](/tests/assertions_operators.yml).
 
+# Write and run your first test suite 
+
+To try to understand how the Venom works, we will create and run a first testsuite together.
+
+The first test that we will do, in this testsuite, is to test whether the site we want to test (a public REST API, OVHcloud API for example):
```
Comment (richardlt) [inline README.md:619]:
```suggestion
The first assertions that we will do, in this testsuite, are to check whether the site we want to test (a public REST API, OVHcloud API for example):
```

File: README.md [hunk 12]
```diff
@@ -472,13 +612,71 @@ Supported operators are `and`, `or` and `xor`.
...
+- is accessible (respond with a 200 status code)
+- respond quickly (less than 1 second)
+- if its answer is not empty
```
Comment (richardlt) [inline README.md:622]:
```suggestion
```

File: README.md [hunk 13]
```diff
@@ -472,13 +612,71 @@ Supported operators are `and`, `or` and `xor`.
...
+- if the expected elements are in its response (which is in JSON format)
```
Comment (richardlt) [inline README.md:623]:
```suggestion
- returns a valid response (which is in JSON format)
```

File: README.md (hunk 9 above)
Comment (scraly) [reply README.md]: done :)

File: README.md (hunk 12 above)
Comment (scraly) [reply README.md:605]: Avec l'assertion "result.bodyjson ShouldContainKey apis" je regarde si la réponse n'est pas vide car je regarde si elle contient "apis"

Comment (richardlt) [reply README.md:605]: I think the last point is enough to describe what you are trying to achieve.

**Comment by scraly** [comment]
I fixed all the requested changes 😉

**Comment by scraly** [comment]
@fsamin @richardlt is it OK for you now? :)

**Review by fsamin — APPROVED** [review]

**Review by richardlt — APPROVED** [review]

**Pull request merged**